prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -158,21 +158,21 @@ void rxr_tx_entry_release(struct rxr_op_entry *tx_entry)
 		rxr_pkt_entry_release_tx(tx_entry->ep, pkt_entry);
 	}
 
-	if (tx_entry->rxr_flags & RXR_OP_ENTRY_QUEUED_RNR)
+	if (tx_entry->rxr_flags & efa_rdm_ope_QUEUED_RNR)
 		dlist_remove(&tx_entry->queued_rnr_entry);
 
-	if (tx_entry->rxr_flags & RXR_OP_ENTRY_QUEUED_CTRL)
+	if (tx_entry->rxr_flags & efa_rdm_ope_QUEUED_CTRL)
```

Comment:
fix case

Reply:
there are several _QUEUED_RNR and _QUEUED_CTRL throughout.

Reply:
Looks like a case of case-insensitive search and replace

Reply:
Fixed